Automattic/wp-calypso
# Add option to update wpcom contact email from domain contact page

#### Changes proposed in this Pull Request

Optionally, allow updating the WordPress.com contact email when updating the domain contact info. Note the addition of the checkbox below the email field. This checkbox is disabled and unchecked if the email entered is identical to the WPCOM contact email address. Otherwise, checking it when submitting the contact info will also set the WPCOM contact email address to the same address as the one for the domain contact info.

Note that the link in the WPCOM verification email must still be clicked to finalize the change. The change to the WPCOM email can be undone from the /me/account page just like a regular WPCOM contact email update.

![2021-05-23 14 12 29](https://user-images.githubusercontent.com/1379730/119290262-2fad7300-bc1a-11eb-8a69-a5327835f6e7.gif)

#### Testing instructions

Go to the edit contact page for a domain.
Make sure the checkbox to update the My Account email is available.
When setting the email field to the same value as the WPCOM email, make sure that the checkbox is disabled.
Set the email field to some new value and save the contact info.
Make sure that the domain info is updated.
You should get a confirmation email asking you to confirm the new WPCOM email address.
